util/src/test/java/org/datacommons/util/McfCheckerTest.java

```diff
@@ -271,7 +271,9 @@ public void checkSVObs() throws IOException, InterruptedException {
             + "value: \"DataSuppressed\"\n"
             + "observationAbout: dcid:geoId/SF\n"
             + "observationDate: \"2020\"\n";
-    assertTrue(success(mcf));
+    // This should only succeed when we allow not-a-number SVObs values
```

Comment:
Should we have a focussed unit test in McfCheckerTest that exercises Sanity_SVObs_Value_NotANumber ?